cool-demos/conditional-blocks.html

```diff
@@ -0,0 +1,452 @@
+                    outline: none !important;
+                    box-shadow: 0 0 0 3px #b4d7ff;
+                }
+
+                .mce-content-body *[contentEditable=false] *[contentEditable=true]:focus {
+                    outline: none;
+                }
+
+                .mce-content-body *[contentEditable=false] *[contentEditable=true]:hover {
+                    outline: none;
+                }
+
+                body {
+                    max-width: 600px;
+                    margin: 2rem auto;
+                }
+
+                a {
+                    color: #2980b9;
+                }
+
+                conditional-block {
+                    margin: 1rem -6px;
+                }
+            `,
+        });
+
+        // Our custom function for setting up the Web Component.
+        // Read more about web components here:
+        // https://developer.mozilla.org/en-US/docs/Web/Web_Components
+        const setup = (win, doc, editor) => {
+            // the shadow root gets it's HTML content from the template element.
+            // We do not need to inject the template element into the content,
+            // we can simply create it in memory and attach it to the shadow root
+            const template = doc.createElement("template");
+
+            template.innerHTML = `
+                <style>
+                    /* The host selector targets the shadow DOM host element
+                     * https://developer.mozilla.org/en-US/docs/Web/CSS/:host()
+                     */
+                    :host {
+                        display: block; /* Required to get block behavior inside TinyMCE */
+                        background-color: rgba(240, 210, 140, .20);
+                        border-radius: 6px;
+                    }
+
+                    header {
+                        display: flex;
+                        padding: 4px 6px;
+                        margin: 0;
+                        background-color: rgba(240, 210, 140, .20);
+                        border-radius: 6px 6px 0 0;
+                    }
+
+                    header p {
+                        margin: 0;
+                        line-height: 24px;
+                        font-size: 14px;
+                        color: #B7974C;
+                    }
+
+                    header > svg {
+                        fill: #B7974C;
+                        margin-right: 6px;
+                    }
+
+                    span#property {
+                        font-weight: bold;
+                    }
+
+                    span#value {
+                        font-weight: bold;
+                    }
+
+                    button {
+                        background: rgba(240, 210, 140, .5);
+                        border: 0;
+                        outline: 0;
+                        -webkit-tap-highlight-color: rgba(0,0,0,0);
+                        -webkit-user-select: none;
```

Comment:
Would `user-select: none` be better here? Otherwise this won't work on other browsers.